### PR TITLE
Odst 105 admins cc dashboard page

### PR DESCRIPTION
**Pull request by BrendenMcGrew**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "graphql": "16.3.0",
     "graphql-config": "^4.1.0",
     "jwt-decode": "^3.1.2",
+    "npm": "^8.5.5",
```
Comment (ericcater) [inline package.json:50]: Probably shouldn't be here

**Review by ericcater — CHANGES_REQUESTED** [review]

**Review by ericcater — APPROVED** [review]

**Comment by yurisim** [comment]
image
Don't spam everyone like this please.

**Comment by yurisim** [comment]
image

Please just use a stylized mat-card, this isn't maintainable.

If you ever notice you duplicate styles, use a apply directive. This is hard to read,
https://tailwindcss.com/docs/reusing-styles

**Comment by yurisim** [comment]
image
Don't use such a generic name

**Comment by yurisim** [comment]
image

the query itself belongs in the servicer, don't call apollo in the component.ts.

However, this is correct, call the other shit in the backend.

**Comment by yurisim** [comment]
image

- [ ] Top margins looks REALLY bad, they are nearly nonexistent

**Comment by yurisim** [comment]
Clicking on these,

image

Should get you to the right pages. It should be active by now in dev

**Review by yurisim — CHANGES_REQUESTED** [review]
please resolve comments

**Comment by yurisim** [comment]
Edges don't look right,
image

image

**Review by ericcater — APPROVED** [review]

**Comment by yurisim** [comment]
image

- [ ] Some elements are left aligned at random intervals, check your break points.
- [ ] Keep verbiage consistent, "Due" for "Unresolved?"

**Review by pmeskridge — APPROVED** [review]

File: apps/ods/src/app/dashboard/dashboard.component.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+.card {
```
Comment (Mikehickey7) [inline apps/ods/src/app/dashboard/dashboard.component.scss:5]: Put it in line ten and instead of using scss

File: package.json [hunk 3]
```diff
@@ -38,7 +38,7 @@
     "@nestjs/jwt": "^8.0.0",
     "@nestjs/passport": "^8.0.1",
     "@nestjs/platform-express": "^8.4.2",
-    "@nrwl/angular": "13.9.4",
+    "@nrwl/angular": "13.9.6",
```
Comment (Mikehickey7) [inline package.json:41]: Did you mean to upgrade?

**Review by Mikehickey7 — APPROVED** [review]
looks good

File: package.json (hunk 3 above)
Comment (BrendenMcGrew) [reply package.json:41]: Yes

**Pull request merged**